lib/redis/RedisSubscriber.js

```diff
@@ -1,95 +1,91 @@
-import { Strategy } from '../constants';
-import { getProcessor } from '../processors';
-import { _ } from 'meteor/underscore';
-import { Meteor } from 'meteor/meteor';
-import extractIdsFromSelector from '../utils/extractIdsFromSelector';
-import RedisSubscriptionManager from './RedisSubscriptionManager';
-import syntheticProcessor from '../processors/synthetic';
-import getDedicatedChannel from '../utils/getDedicatedChannel';
+import RedisPipe, { Strategy } from "../constants";
+import { getProcessor } from "../processors";
+import { Meteor } from "meteor/meteor";
+import extractIdsFromSelector from "../utils/extractIdsFromSelector";
+import RedisSubscriptionManager from "./RedisSubscriptionManager";
+import syntheticProcessor from "../processors/synthetic";
+import getDedicatedChannel from "../utils/getDedicatedChannel";
 
 export default class RedisSubscriber {
-    /**
-     * @param observableCollection
-     * @param strategy
-     */
-    constructor(observableCollection, strategy) {
-        this.observableCollection = observableCollection;
-        this.strategy = strategy;
-        this.processor = getProcessor(strategy);
+  /**
+   * @param observableCollection
+   * @param strategy
+   */
+  constructor(observableCollection, strategy) {
+    this.observableCollection = observableCollection;
+    this.strategy = strategy;
+    this.processor = getProcessor(strategy);
 
```

Comment:
why did you modify the indentation?

Reply:
I just had conflicting prettier settings on my initial save so I had to go back and fix the formatting, must have missed this spot. I'll fix the formatting